webui/react/src/components/ExperimentChart.tsx

```diff
@@ -0,0 +1,140 @@
+import { Select } from 'antd';
+import { SelectValue } from 'antd/es/select';
+import Plotly, { PlotData } from 'plotly.js/lib/core';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+
+import { ValidationHistory } from 'types';
+import { clone } from 'utils/data';
+import { formatDatetime } from 'utils/date';
+import { capitalize, generateAlphaNumeric } from 'utils/string';
+
+import Section from './Section';
+import SelectFilter from './SelectFilter';
+
+const { Option } = Select;
+
+interface Props {
+  id?: string;
+  startTime?: string;
+  validationMetric?: string;
+  validationHistory?: ValidationHistory[];
+}
+
+enum Scale {
+  Linear = 'linear',
+  Log = 'log',
+}
+
+type PlotArguments = [
+  string,
+  Partial<PlotData>[],
+  Partial<Plotly.Layout>,
+  Partial<Plotly.Config>,
+];
+
+const defaultLayout: Partial<Plotly.Layout> = {
+  xaxis: {
+    hoverformat: '',
+    title: 'Elapsed Time (seconds)',
+  },
+  yaxis: {
+    title: 'Metric Value',
+    type: Scale.Linear,
+  },
+};
+
+const defaultConfig: Partial<Plotly.Config> = {
+  responsive: true,
+};
+
+const ExperimentChart: React.FC<Props> = ({ validationMetric, ...props }: Props) => {
+  const id = props.id ? props.id : generateAlphaNumeric();
+  const titleDetail = validationMetric ? ` (${validationMetric})` : '';
+  const title = `Best Validation Metric${titleDetail}`;
+  const [ scale, setScale ] = useState<Scale>(Scale.Linear);
+  const [ isRendered, setIsRendered ] = useState(false);
+
+  const data: Partial<PlotData>[] = useMemo(() => {
+    if (!props.startTime || !props.validationHistory) return [];
+
+    const startTime = new Date(props.startTime).getTime();
+    const textData: string[] = [];
+    const xData: number[] = [];
+    const yData: number[] = [];
+
+    props.validationHistory.forEach(validation => {
+      const endTime = new Date(validation.endTime).getTime();
+      const x = (endTime - startTime) / 1000;
+      const y = validation.validationError;
+      const text = [
+        `Trial ${validation.trialId}`,
+        `Time: ${formatDatetime(validation.endTime)}`,
```

Comment:
IMO we can do without time and trial here. the time is gonna be similar on a bunch of metrics (if I'm not mistaken because it's not exactly the time that metric was generated but when it was submitted as a step?)

![Screen Capture_select-area_20200728163510](https://user-images.githubusercontent.com/12127420/88739897-90e6e180-d0f0-11ea-855d-6ae8cadc52e0.png)

Reply:
I think we can drop the `Time:` label. I was hoping to get rid of the bottom black hover for the time but could not figure out if that's even an option.